app/components/timeline_entry/component.rb

```diff
@@ -199,5 +199,21 @@ def stage_changed_vars
           ).strip,
       }
     end
+
+    def application_declined_vars
+      declined =
+        if timeline_event.application_form.declined_at?
+          render(
+            StatusTag::Component.new(
+              "declined",
+              class_context: "timeline-event",
+            ),
+          ).strip
+        else
+          ""
+        end
+
+      { declined: }
```

Comment:
It looks like this `declined` variable isn't used anywhere so we can probably delete it.